.github/workflows/integ-tests.yml

```diff
@@ -0,0 +1,211 @@
+name: Integ Tests
+
+# TODO set this up to gate new releases of Syne Tune
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+  pull_request: # remove before merging, this is for testing
+    branches:   # remove before merging, this is for testing
+      - main    # remove before merging, this is for testing
+
+permissions:
+  id-token: write # This is required for requesting the JWT
+  contents: read  # This is required for actions/checkout
+
+env:
+  AWS_DEFAULT_REGION: us-west-2
+  AWS_ROLE: ${{ secrets.PROD_AWS_INTEG_TEST_ROLE_ARN }}
+
+jobs:
+  stop_all_training_jobs: 
+    runs-on: ubuntu-latest
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.PROD_AWS_INTEG_TEST_ROLE_ARN }}
+          role-session-name: integtestsession
+          aws-region: ${{ env.AWS_DEFAULT_REGION }}
+      - name: Stop all left-over training jobs
+        run: |
+          aws sagemaker list-training-jobs --status-equals InProgress > running_jobs.json
```

Comment:
This should probably be more inclusive, for instance it should probably also include "Starting".

Reply:
I spent a few minutes and I haven't found the syntax for how to match two statues from the AWS CLI, do you know what it would look like? In my testing, I haven't found it to be a problem, since we have enough of a limit in the account to have some buffer

Reply:
Just checked and it cannot be done indeed.